content/en/monitors/notify/_index.md

```diff
@@ -130,7 +130,7 @@ An [event][16] is created anytime a monitor is created, modified, silenced, or d
 
 ### Edit restrictions
 
-If changes are restricted, only the monitor's creator or an administrator can change the monitor. Changes include any updates to the monitor definition and muting for any amount of time.
+If changes are restricted, only the monitor's creator or a user with the User Access Manage permission can change the monitor. Changes include any updates to the monitor definition and muting for any amount of time.
```

Comment:
Sorry for the churn but the use-case of Edit-restricted monitors changed in the last couple of weeks after Twilio didn't like it. Instead of User Access Manage, it's back to "admin" (until Jan ~18) and then Org Management afterwards.